components/dashboard/src/start/StartPage.tsx

```diff
@@ -105,6 +108,7 @@ export function StartPage(props: StartPageProps) {
                         error || phase === StartPhase.Stopped || phase === StartPhase.IdeReady ? "" : "animate-bounce"
                     }`}
                 />
+                {!isDataOps && <span className="block mt-2 text-gray-600 dark:text-gray-400">Gitpod Classic</span>}
                 <Heading2 className="mt-8">{title}</Heading2>
```

Comment:
let's use the podkit color to be consistent with the Figma design:

```suggestion
                {!isDataOps && <span className="block mt-2 text-pk-content-secondary">Gitpod Classic</span>}
                <Heading2 className="mt-8">{title}</Heading2>
```

Reply:
Damn, VS Code's GitHub extension really loves to mess up suggestion diffs. This was meant for line 111, so please don't apply as-is.

Reply:
Resolved in 657d73f